CMakeLists.txt

```
@@ -270,6 +272,22 @@ endif ()
 set(CMAKE_THREAD_PREFER_PTHREAD True)
 find_package(Threads REQUIRED)
 
+if (USE_TRACY)
+    include(FetchContent)
+    add_compile_definitions(TRACY_ENABLE TRACY_ON_DEMAND)
+
+    FetchContent_Declare(
+        tracy
+        GIT_REPOSITORY https://github.com/wolfpld/tracy.git
+        GIT_TAG master
+        GIT_SHALLOW TRUE
+        GIT_PROGRESS TRUE)
+
+    FetchContent_MakeAvailable(tracy)
+
+    target_compile_options(TracyClient PRIVATE "-Wno-everything")
```

msvc doen't have such flag
```suggestion
    if(NOT MSVC)
        target_compile_options(TracyClient PRIVATE "-Wno-everything")
    endif()
```